slides/02/02.md

```diff
@@ -29,7 +29,7 @@ $$→w = (⇉X^T⇉X)^{-1}⇉X^T→t.$$
 
 Assume we want to predict a $t ∈ ℝ$ for a given $x ∈ ℝ$. If we train
 the linear regression with “raw” input vectors $→x = (x)$, only straight lines
-could be modeled.
+could be modelled.
```

Comment:
`modeled` is also correct.

Reply:
I go for US spelling when the US and British variants differ (I believe you see "modeling" and "modeled"  more often in papers). I will revert it once merged.